From dd9daa56a7e6f250a70e119502a6481d4089cec3 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Tue, 28 Apr 2020 09:34:01 +0800
Subject: [PATCH] ARM: dts: rk312x: delete critical trip for thermal

rk312x use virtual tsadc, there is no need poweroff when overheat,
just limit the frequency.

Change-Id: Id43c76ca6876a5c38ced9e0c07511ee57dd53691
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 37924f4f4a3f..29188e5a7e6b 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -291,11 +291,6 @@
 					hysteresis = <2000>;
 					type = "passive";
 				};
-				soc_crit: soc-crit {
-					temperature = <115000>;
-					hysteresis = <2000>;
-					type = "critical";
-				};
 			};
 
 			cooling-maps {
-- 
2.35.3

